### DISCUSSION TITLE
File hierarchy

### DISCUSSION THREAD

**Discussion by Stalinosmj** [jude7733/ZeroLatency#1] [Q&A] [answered]
Explain the files and contents

**Comment by jude7733** [comment, marked as answer]
/src contains react files (source code)
/models for storing blender 3d models
/components holds custom components(widgets)
entry point index.html
DO NOT change configs (vite, github workspace, package.json, yarn.lock)
/src/App.jsx is parent

**Reply by jude7733** [reply]
https://sketchfab.com/3d-models?features=downloadable&sort_by=-likeCount

**Reply by Stalinosmj** [reply]
Da /models enn oru folder undagande, ath evde?

**Reply by jude7733** [reply]
Indakiko ,, ipo onum ilala

**Comment by jude7733** [comment]
https://vitejs.dev/guide/
https://docs.pmnd.rs/react-three-fiber/getting-started/introduction

**Comment by jude7733** [comment]
install nodejs
go to project folder
"yarn" command installs dependencies
"yarn run dev" starts dev server
